media/js/src/utils.js

```diff
@@ -814,6 +814,22 @@ const getTerms = function(annotations) {
     return terms;
 };
 
+const setAriaLabel = function(type, id, title,) {
+    if (type === 'video') {
+        document.getElementById(`video-${id}`).setAttribute(
+            'alt', `Annotation titled ${title}`);
+    } else if (type === 'audio') {
+        document.getElementById(`audio-${id}`).setAttribute(
+            'alt', `Annotation titled ${title}`);
+    } else if (type === 'unknown') {
+        document.getElementById(`unknown-${id}`).setAttribute(
+            'alt', `Annotation titled ${title}`);
+    } else {
+        document.getElementById(`map-${id}`).setAttribute(
+            'aria-label', `Annotation titled ${title}`);
```

Comment:
This way of updating things can work.. but this is a sort of imperative, "jQuery"-way of updating things, that is prone to breaking with future changes.

A more solid, declarative approach would be better. Since we happen to already be in React here, we can do that. The dom element's alt text can be declared more directly in the parent component - GridAsset.jsx.

The parent component already has a `selectedAnnotation` state variable, and we can use that to determine what the element's alt text should be in a more direct fashion. And also, any other attribute we want to set. Altering the elements after they've been rendered, with setAttribute, is unnecessary.